course_contents/2_intro_to_python/notes/04. Destructuring.md

```diff
@@ -1,13 +1,11 @@
 ### Unpacking
-- unpacking is used to assign values from a collection to variables in a single statement
+- unpacking is used to assign values from an iterable to variables in a single statement
```

Comment:
I'd bring back `collection` since we decided at the top that we're not going to talk about `iterable`

Reply:
So we are going with a collection instead of an iterable at all places where it appears?
I am asking this just to make sure I understood you properly before I get to changing it

Reply:
Resolving this and other conversations including this discussion since we agreed to go with iterable